new/INS1005Maaqud.xml

```diff
@@ -0,0 +1,94 @@
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church religiousSite" subtype="institution">                    
+                            <placeName xml:id="n1" xml:lang="gez">መዓቁዲ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፡</placeName>
+                            <placeName corresp="#n1" xml:lang="en">Maʿāquddi Yoḥannǝs Walda Nagʷadgʷād</placeName>
+                            <placeName xml:id="n2" xml:lang="gez">ዮሐንስ፡ መዓቁዲ፡</placeName>
+                            <placeName corresp="#n2" xml:lang="en">Yoḥannǝs Maʿāquddi</placeName>
+                    <placeName xml:id="n3" xml:lang="gez">ዮሐንስ፡ መዓቁዲ፡ ደብረ፡ ከርቤ፡ </placeName>
+                    <placeName corresp="#n3" xml:lang="en">Yoḥannǝs Maʿāquddi Dabra Karbe </placeName>
+                    <placeName xml:id="n4" xml:lang="gez">ደብረ፡ ከርቤ፡ ዮሐንስ፡ መዓቁዲ፡</placeName>
+                    <placeName corresp="#n4" xml:lang="en">Yoḥannǝs Maʿāquddi Dabra Karbe</placeName>
+                            <placeName xml:id="n5" xml:lang="gez">ዮሐንስ፡ ዓዲ፡ አይኑ፡</placeName>               
+                            <placeName corresp="#n5" xml:lang="en">Yoḥannǝs ʿĀddi ʿĀynu</placeName>
+                            
+                            <country ref="LOC3010Ethiop"></country>
+                            <region notBefore="1991" type="RegionalState" ref="LOC5932Tegray">The regional State of Tǝgrāy</region>  
+                    <settlement ref="LOC6653EastTegray" type="zone" when="2023-11-15">East Tǝgrāy zone</settlement>
+                    <settlement ref="LOC3826Hawzen" type="warada" when="2023-11-15"> waradā  Ḥawzen</settlement>
+                    <settlement when="2023-11-15" ref="LOC7408Maykwiha" type="tabiya">ṭabiyā Māy Kʷiḥā</settlement>
+                    <settlement when="2023-11-15" ref="LOC4358Maaqud" type="qushat">qušat Maʿāquddi</settlement>
+                            <location>
+                                <geo>13.849072 39.447303</geo>
+                                <height unit="m">2405.0</height>
+                            </location>
+                            <note>
+                                <date type="foundation" notBefore="1400" notAfter="1499">Possibly 15th century</date>
+                                <ab type="history">Dabr. Rock-hewn church well-known for its frescos depicting <persName ref="PRS4015Fasilada">King Fasiladas</persName>.
+                                </ab>
+                                <ab type="tabot">
+                                    <measure quantity="2"></measure>                                   
+                                    <persName ref="PRS5695John" type="tabot">Yoḥannǝs Walda Nagʷadgʷād</persName>, 
+                                    <persName ref="PRS6819Mary" type="tabot">Mary</persName>
+                                    <persName ref="PRS10776Kidan" type="tabot">the Covenant of Mercy</persName>
```

Comment:
`PRS10776Kidan `does not exist, we do not have PRS records for non-persons. You should link to the keyword `CovenantMercy`
`<ref type="authFile" corresp="CovenantMercy">Kidāna Mǝḥrat<ref>`

Reply:
ok

Reply:
not corrected yet?

Reply:
sorry, one update was forgotten.